src/WPF/ArcGISRuntime.WPF.Viewer/Samples/Map/BrowseBuildingFloors/BrowseBuildingFloors.xaml.cs

```diff
@@ -0,0 +1,94 @@
+﻿// Copyright 2022 Esri.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at: http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific
+// language governing permissions and limitations under the License.
+
+using Esri.ArcGISRuntime;
+using Esri.ArcGISRuntime.Mapping;
+using Esri.ArcGISRuntime.Mapping.Floor;
+using Esri.ArcGISRuntime.Portal;
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using System.Windows;
+using System.Windows.Controls;
+
+namespace ArcGISRuntime.WPF.Samples.BrowseBuildingFloors
+{
+    [ArcGISRuntime.Samples.Shared.Attributes.Sample(
+        name: "Browse building floors",
+        category: "Map",
+        description: "Display and browse through building floors from a floor-aware web map.",
+        instructions: "Use the combo box to browse different floor levels in the facility. Only the selected floor will be displayed.",
+        tags: new[] { "building", "facility", "floor", "floor-aware", "floors", "ground floor", "indoor", "level", "site", "story" })]
+    public partial class BrowseBuildingFloors
+    {
+        private const string _floorData = "https://ess.maps.arcgis.com/home/item.html?id=f133a698536f44c8884ad81f80b6cfc7";
```

Comment:
I think this might be confusing, since the constant is holding a string with a URL, not floor data.

Since the URL isn't referenced anywhere else, the simplest solution is probably to put the string literal in the URL constructor on line 43.